Documentation/ceph-cluster-crd.md

```diff
@@ -216,6 +218,7 @@ For more details on the mons and when to choose a number other than `3`, see the
   * `config`: Config settings applied to all OSDs on the node unless overridden by `devices`. See the [config settings](#osd-configuration-settings) below.
   * [storage selection settings](#storage-selection-settings)
   * [Storage Class Device Sets](#storage-class-device-sets)
+  * `onlyApplySpecificOSDPlacement`: `false` or `true`, In case of false, for OSD, will merge nodeAffinity of the placement.All() with placement.osd(in case of non-PVCs) or storageClassDeviceSets.placement(in case of PVCs), for prepareOSD will merge nodeAffinity of the placement.All() with placement.prepareosd(in case of non-PVCs) or storageClassDeviceSets.preparePlacement(in case of PVCs) vice-versa when true.
```

Comment:
I would suggest using an easier explanation as mentioned in the [comment](https://github.com/rook/rook/pull/8381#discussion_r678411565).

Reply:
added.